rio/placer/placers_darwin.go

```diff
@@ -0,0 +1,124 @@
+// +build linux
+package placer
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/spacemonkeygo/errors/try"
+	"go.polydawn.net/repeatr/lib/fs"
+	"go.polydawn.net/repeatr/lib/fspatch"
+	"go.polydawn.net/repeatr/rio"
+)
+
+var _ rio.Placer = CopyingPlacer
+
+func CopyingPlacer(srcBasePath, destBasePath string, _ bool, bareMount bool) rio.Emplacement {
```

Comment:
if this doesn't have any refs mac be hatin on, can we move it to a `placers_allplat.go` file? (or some name like that)